File: felipe/Problema0160/problema0160.java
import java.util.Scanner;

public class problema0160{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Digite o número de horas extras: ");
        double horasExtras = Double.parseDouble(input.next());

        System.out.print("Número de horas faltadas: ");
        double horasFaltadas = Double.parseDouble(input.next());

        double h = horasExtras - 2/3 * horasFaltadas;

        if(h > 2400){
            System.out.println("Gratificação de R$500,00");
        } else {
            if(h > 1800){
                System.out.println("Gratificação de R$400,00");
            } else {
                if(h > 1200){
                    System.out.println("Gratificação de R$300,00");
                } else {
                    if(h > 600){
                        System.out.println("Gratificação de R$200,00");
                    } else {
                        System.out.println("Gratificação de R$100,00");
                    }
                }
            }
        }
    }
}

File: felipe/Problema0160/problema0160.py
horasExtras = float(input("Digite o número de horas extras: "))
horasFaltadas = float(input("Número de horas faltadas: "))

h = horasExtras - 2/3 * horasFaltadas

if h > 2400:
    print("Gratificação de R$500,00")
elif h > 1800:
    print("Gratificação de R$400,00")
elif h > 1200:
    print("Gratificação de R$300,00")
elif h > 600:
    print("Gratificação de R$200,00")
else:
    print("Gratificação de R$100,00")